### PR TITLE
Feature/http timeouts

### PR DESCRIPTION
**Pull request by peteclark-ft**

### CODE REVIEW

File: annotations/annotations_api_test.go [hunk 1]
```diff
@@ -2,12 +2,15 @@ package annotations
 
 import (
 	"context"
+	"net"
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	"time"
 
 	"github.com/Financial-Times/go-ft-http-transport/transport"
```
Comment (galiri) [inline annotations/annotations_api_test.go]: shouldn't we use go-ft-http/fthttp lib here?

File: concept/search_api_test.go [hunk 2]
```diff
@@ -4,13 +4,16 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"net"
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	"time"
 
 	"github.com/Financial-Times/go-ft-http-transport/transport"
```
Comment (galiri) [inline concept/search_api_test.go]: shouldn't we use go-ft-http/fthttp lib here with timeout set to 0?

File: annotations/annotations_rw_test.go [hunk 3]
```diff
@@ -207,6 +210,22 @@ func TestWriteMissingTID(t *testing.T) {
...
 }
 
+func TestRWTimeout(t *testing.T) {
+	r := vestigo.NewRouter()
+	r.Put("/drafts/content/:uuid/annotations", func(w http.ResponseWriter, r *http.Request) {
+		time.Sleep(500 * time.Millisecond)
+	})
+
+	s := httptest.NewServer(r)
+	rw := NewRW(testClient, s.URL)
```
Comment (galiri) [inline annotations/annotations_rw_test.go]: same question as in other test files for the testClient

**Comment by erdema-ft** [comment]
@peteclark-ft I've also updated the us staging configmap, just fyi;

`draft-annotations-api-timeout: 8s`

**Pull request merged**